packages/manager/modules/vps/src/dashboard/vps-dashboard.constants.js

```diff
@@ -12,10 +12,25 @@ export const NEW_RANGE_VERSION = '2019v1';
 export const VPS_2014_AUTO_MIGRATION_DATE = '18/07/2020';
 export const VPS_2014_EXPIRY_DATE = '27/07/2020';
 
+export const RTM_WARNING_VALUE = {
+  min: 50,
+  max: 80,
+};
+
+export const GUIDE_RTM =
+  'https://docs.ovh.com/gb/en/vps/getting-started-vps/#enablert';
```

Comment:
Only en_GB value ?

Reply:
yes, we have only this guide available for all :/